CommonSelectors.py

```diff
@@ -394,14 +504,126 @@ def ll_2j(lep_flav='both'):
 
 def Zll_2j(lep_flav='both'):
     return  Cut(
-        name = 'Zll_2j_lepflav_'+lep_flav,
+        name = 'Zll_2j_lepflav_'+lepflav,
```

Comment:
This needs to be `lep_flav`, could you change.

Reply:
Sorry for the delay.. Updated the typo, thanks!